WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController+InformativeDialog.swift

```diff
@@ -5,7 +5,7 @@ extension GutenbergViewController {
 
     enum InfoDialog {
         static let message = NSLocalizedString(
-            "This post uses the block editor, which is the default editor for new posts. To enable the classic editor, go to Me > App Settings.",
+            "This post uses the block editor, which is the default editor for new posts. To enable the classic editor, go to Site Settings.",
```

Comment:
Let's update the copy to match the latest agreed over at the Android side PR: `You're currently using the block editor to create new [posts/pages]. Want to use the classic editor by default? Head to Site Settings.` https://github.com/wordpress-mobile/WordPress-Android/pull/10254#issuecomment-515334567

Reply:
Updated!